test/functional/apps/discover/group1/_discover.ts

```diff
@@ -112,7 +111,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         );
       });
 
-      it('should show correct initial chart interval of Auto', async function () {
+      // FLAKY: https://github.com/elastic/kibana/issues/146223
+      it.skip('should show correct initial chart interval of Auto', async function () {
```

Comment:
I moved the `.skip` to the actual failing test so the whole suite isn't skipped, and so my new test would run.